project-template/index.template.js

```diff
@@ -1 +1,17 @@
-console.log('I am the bot.');
+const { registerModule } = require('nay');
+
+function onReceive(sender) {
+  // TODO
+}
+
+function onEnable() {
+  // TODO
+}
+
+function onDisable() {
+  // TODO
+}
+
+const module = { onReceive, onEnable, onDisable };
+
+registerModule('/command', module);
```

Comment:
Just a little point: I think we could move this code to the top of a file like that:

```javascript
const { registerModule } = require('nay');
registerModule('/command', { onReceive, onEnable, onDisable });

function onEnable() {}
function onReceive(sender) {}
function onDisable() {}
```

Reply:
Yes, functions are hoisted, but I think we shouldn't use them before definition.
```js
// Both examples below are identical

// Non-hoisted version
function sayHello() {
  console.log('Hello!');
}
sayHello();

// Hoisted version
sayHello();
function sayHello() {
  console.log('Hello!');
}
```

Output will be `"Hello!"` in both cases, but first version is more natural to read I think.